src/common/snippets/include/snippets/lowered/loop_info.hpp

```diff
@@ -45,7 +45,8 @@ class LoopInfo : public std::enable_shared_from_this<LoopInfo> {
     LoopInfo(size_t work_amount,
              size_t increment,
              const std::vector<LoopPort>& entries,
-             const std::vector<LoopPort>& exits);
+             const std::vector<LoopPort>& exits,
+             bool is_parallel = false);
```

Comment:
Probably it is better to make it non default to be consistent with other APIs that require it explicitly, e.g. UnifiedLoopInfo

Reply:
Good point, thank you! I made it none-default, as well as in LoopBegin/LoopEnd nodes